proxy/backend/type/hbase/src/main/java/org/apache/shardingsphere/proxy/backend/hbase/context/HBaseRegionWarmUpContext.java

```diff
@@ -72,6 +74,24 @@ public void submitWarmUpTask(final String tableName, final HBaseCluster hbaseClu
         executorManager.submit(() -> loadRegionInfo(tableName, hbaseCluster));
     }
     
+    /**
+     * load one table region info.
```

Comment:
1. The first letter should be upper-case
2. Javadoc and @param should keep a blank line